faction.h

```diff
@@ -203,7 +205,7 @@ class Faction : public AListElem
 	//
 	AList extraPlayers;
 	AList errors;
-	AList events;
+	std::vector<IEvent*> events_;
```

Comment:
Maybe `std::list` is better here as the number of events will vary

Reply:
On a modern processor, vector will outperform list in almost every case.